test/data/sandbox/custom-worker/custom_test.worker.js

```diff
@@ -0,0 +1,7 @@
+Feature('Custom Workers');
+
+Scenario('say custom something', (I) => {
+  I.say('Hello Workers');
+  I.sayCustomMessage();
+});
+
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Too many blank lines at the end of file. Max of 0 allowed.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4759786)